.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
```

Comment:
The comment `Unless a later match takes precedence` is not clear. I think it should be `When several patterns match the same file, the last one will take precedence`.